compute-at-edge.json

```diff
@@ -0,0 +1,26 @@
+{
+	"llvm-target":   "wasm32-unknown-wasi",
```

Comment:
We can probably use `"inherits":"wasi",` here and just override the emulator line.

Reply:
nice!  I did not know about this, it makes things so much cleaner!  Did that in 9341fec.